content/pt-br/docs/concepts/security/windows-security.md

```diff
@@ -0,0 +1,56 @@
+---
+reviewers:
+  -
+title: Segurança para Nós Windows
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+Esta página descreve considerações de segurança e boas práticas específicas para o sistema operacional Windows.
+
+<!-- body -->
+
+## Proteção para dados Secret nos Nós
+
+No Windows, os dados do Secret são escritos em texto claro no Nó local do
+armazenamento (em comparação ao uso de tmpfs / in-memory filesystems no Linux). Como um cluster
+operador, você deve tomar as duas medidas adicionais a seguir:
+
+1. Use arquivos ACLs para assegurar a localização do arquivo Secrets.
+2. Aplicar criptografia à nível de volume usando
+   [BitLocker](https://docs.microsoft.com/windows/security/information-protection/bitlocker/bitlocker-how-to-deploy-on-windows-server).
+
+## Usuários dos Contêineres
+
+[RunAsUsername](/docs/tasks/configure-pod-container/configure-runasusername)
+pode ser especificado para Pods com Windows ou contêiner para executar os processos do contêiner como usuário específico. Isto é aproximadamente equivalente a
```

Comment:
This sentence needs to be reviewed. It's not really delivering the original meaning (a pod or a container with Windows can have this field specified).

Reply:
Hi, I did a minor change, but I don't know if it's right now.